### DISCUSSION TITLE
Question about screenshots

### DISCUSSION THREAD

**Discussion by frakman1** [bensadeh/tailspin#63] [Q&A] [answered]
Nice tool. I look forward to using it now.

I really love the way your README file is formatted. I like how you use HTML tags to get around some of Github Markdown's constraints like centering your images.

I was curious though, how did you get the example console output screenshots to look that way? They have a rounded outline and a shadow around them. I think I've seen a tool that did that once but can't remember what it is.

Cheers.

**Comment by bensadeh** [comment, marked as answer]
Hi again @frakman1 , glad you're enjoying tailspin and the readme.

For the rounded outline, use iTerm2 without without header. And for the shadows, I just use macOS's builtin tool to take screenshots of windows. It's Cmd + Shift + 4 and then space to change the mode to be the selected window instead of dragging a rectangle.

**Comment by frakman1** [comment]
Thank you! 
For future reference, I was able to remove the title bar here:
`Preferences -> ProfileName -> Window`
![image](https://github.com/bensadeh/tailspin/assets/5826484/955313fc-fa1b-4ec0-a4e6-4737317d1451)

... and hide the scrollbar here:
`Preferences -> Appearance -> Windows -> Hide Scrollbars`
![image](https://github.com/bensadeh/tailspin/assets/5826484/7e18141e-8984-4be1-b166-3afaf675889b)

I just wish hiding the scroll bar was a profile setting instead of a global setting.
